Orange/preprocess/score.py

```diff
@@ -207,7 +207,7 @@ def _gini(D):
     """Gini index of class-distribution matrix"""
     P = D / np.sum(D, axis=0)
     return sum((np.ones(1 if len(D.shape) == 1 else D.shape[1]) - np.sum(np.square(P), axis=0))
-               * 0.5 * np.sum(D, axis=0) / np.sum(D))
+               * np.sum(D, axis=0) / np.sum(D))
```

Comment:
This is out of league. Perhaps merge this PR to fix division by two and make another one with improved formula? Otherwise I'll just copy-paste whatever you'll tell me to. :grinning: